Assets/RingForSpecialSpell.cs

```diff
@@ -0,0 +1,20 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class RingForSpecialSpell : MonoBehaviour {
```

Comment:
Name this maybe in a generic way.
This may not be a specific ring.
Use SpecialSpellIndicator or something maybe better.